src/goose.rs

```diff
@@ -2426,30 +2488,33 @@ mod tests {
 
         // Make a GET request to the mock http server and confirm we get a 200 response.
         assert_eq!(mock_index.times_called(), 0);
-        let goose = user.get("/").await;
+        let goose = user.get("/").await.expect("get returned unexpected error");
         let status = goose.response.unwrap().status();
         assert_eq!(status, 200);
-        assert_eq!(mock_index.times_called(), 1);
         assert_eq!(goose.request.method, GooseMethod::GET);
         assert_eq!(goose.request.name, "/");
         assert_eq!(goose.request.success, true);
         assert_eq!(goose.request.update, false);
         assert_eq!(goose.request.status_code, 200);
+        assert_eq!(mock_index.times_called(), 1);
```

Comment:
This is an intentional change?

Reply:
The original reason for this change was so it was outside of the {} which I temporarily added through this PR. I left the change as it seemed tidy for it to consistently come last in all blocks.